src/core/ddsi/src/ddsi_cdrstream.c

```diff
@@ -996,6 +1656,7 @@ static bool normalize_primarray (char * __restrict data, uint32_t * __restrict o
       *off += 2 * num;
       return true;
     case DDS_OP_VAL_4BY:
+    case DDS_OP_VAL_ENU:
```

Comment:
Can you please double-check that arrays of 64-bit elements do retain 8 byte alignment even in XCDR2? It is the OMG, so I certainly consider such an inconsistency quite believable.

Reply:
No, it's also 4 byte alignment, I've fixed this in 1b3eb65